doc/configuration.rst

```diff
@@ -794,36 +794,93 @@ The skeleton for an environment consists of:
 
 .. code-block:: yaml
 
-   <target-1>:
-     resources:
-       <resources>
-     drivers:
-       <drivers>
-   <target-2>:
-     resources:
-       <resources>
-     drivers:
-       <drivers>
-
-If you have a single target environment, name it "main", the ``get_target``
-function defaults to "main".
+   targets:
+     <target-1>:
+       resources:
+         <resource-1>:
+           <resource-1 parameters>
+         <resource-2>:
+           <resource-2 parameters>
+       drivers:
+         <driver-1>:
+           <driver-1 parameters>
+         <driver-2>: {} # no parameters for driver-2
+     <target-2>:
+       resources:
+         <resources>
+       drivers:
+         <drivers>
+     <more targets>
+   options:
+     <option-1 name>: <value for option-1>
+     <more options>
+   images:
+     <image-1 name>: <absolute or relative path for image-1>
+     <more images>
+   tools:
+     <tool-1 name>: <absolute or relative path for tool-1>
+     <more tools>
+
+If you have a single target in your environment, name it "main", as the
+``get_target`` function defaults to "main".
 
 All the resources and drivers in this chapter have a YAML example snippet which
 can simply be added (at the correct indentation level, one level deeper) to the
 environment configuration.
 
 Exporter Configuration
 ----------------------
-A labgrid exporter is configured by adding resources to groups and exporting
-those groups.
+The exporter is configured using a YAML file with a similar syntax to the
+environment configs used for pytest or instantiating the :any:`Environment`
+object.
```

Comment:
A bit tricky to read here, I would say at least "configured *by* using a YAML file" and "pytest or *by* instantiating" and/or putting "(with a similar syntax to the environment configs used for pytest)" in brackets